ubuntudesign/www.ubuntu.com
# remove the tablet and phone sections and their css

## Done

Removed /phone and /tablet sections

## QA

Ensure that the sections are removed and that there are no bugs related to the change.
